lib/services/dialogflow/lifecycle/es/utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ResponseMessage, WebhookResponse } from '../../types';
+
+// eslint-disable-next-line import/prefer-default-export
+export const addResponseMessage = (res: WebhookResponse, text: string, messages?: ResponseMessage[]) => {
+  res.fulfillmentText = `${res.fulfillmentText} ${text}`.trim();
+
+  if (messages) {
+    res.fulfillmentMessages.push(...messages);
+  } else {
+    res.fulfillmentMessages.push({ text: { text: [text] } });
+  }
```

Comment:
setting a default value for `messages` makes the function body cleaner

```suggestion
export const addResponseMessage = (res: WebhookResponse, text: string, messages: ResponseMessage[] = [{ text: { text: [text] } }]) => {
  res.fulfillmentText = `${res.fulfillmentText} ${text}`.trim();

  res.fulfillmentMessages.push(...messages);
```